### PR TITLE
feat: updates and fixes

### PR DESCRIPTION
**Pull request by nasbench**
This PR adds
- New rules
- Multiple renames and fixes for old rules
- FP fixes found in testing

### CODE REVIEW

File: rules/windows/process_creation/proc_creation_win_bad_opsec_sacrificial_processes.yml [hunk 1]
```diff
@@ -4,7 +4,7 @@ related:
     - id: f5647edc-a7bf-4737-ab50-ef8c60dc3add
       type: obsoletes
 status: experimental
-description: 'Detects attackers using tooling with bad opsec defaults e.g. spawning a sacrificial process to inject a capability into the process without taking into account how the process is normally run, one trivial example of this is using rundll32.exe without arguments as a sacrificial process (default in CS, now highlighted by c2lint), running WerFault without arguments (Kraken - credit am0nsec), and other examples.'
+description: Detects attackers using tooling with bad opsec defaults e.g. spawning a sacrificial process to inject a capability into the process without taking into account how the process is normally run, one trivial example of this is using rundll32.exe without arguments as a sacrificial process (default in CS, now highlighted by c2lint), running WerFault without arguments (Kraken - credit am0nsec), and other examples.
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_bad_opsec_sacrificial_processes.yml:7]:
```suggestion
description: |
    Detects attackers using tooling with bad opsec defaults.
    E.g. spawning a sacrificial process to inject a capability into the process without taking into account how the process is normally run.
    One trivial example of this is using rundll32.exe without arguments as a sacrificial process (default in CS, now highlighted by c2lint), running WerFault without arguments (Kraken - credit am0nsec), and other examples.
```
One trivial  but get 2 in the sentense

File: rules/windows/process_creation/proc_creation_win_powershell_cmdline_reversed_strings.yml [hunk 2]
```diff
@@ -1,7 +1,7 @@
-title: Suspicious PowerShell Cmdline
+title: Potential PowerShell Obfuscation Via Reversed Commands
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_powershell_cmdline_reversed_strings.yml:1]: need update modified

File: rules/windows/process_creation/proc_creation_win_susp_plink_usage.yml [hunk 3]
```diff
@@ -1,5 +1,8 @@
-title: Suspicious Plink Usage RDP Tunneling
+title: Potential RDP Tunneling Via SSH Plink
```
Comment (frack113) [inline rules/windows/process_creation/proc_creation_win_susp_plink_usage.yml:1]: need to update modified

File: rules/windows/process_creation/proc_creation_win_anydesk_execution_from_susp_folders.yml [hunk 4]
```diff
@@ -1,5 +1,8 @@
-title: Use of Anydesk Remote Access Software from Suspicious Folder
+title: Anydesk Execution From Suspicious Folder
```
Comment (phantinuss) [inline rules/windows/process_creation/proc_creation_win_anydesk_execution_from_susp_folders.yml:1]: also needs an updated modified

**Review by frack113 — APPROVED** [review]

**Pull request merged**